### PR TITLE
[provisioning] fixes for production provision/deploy

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: ansible/adrl.yml [hunk 1]
```diff
@@ -1,13 +1,12 @@
 - name: ADRL on CentOS
   hosts: all
-  remote_user: vagrant
+  remote_user: "{{ lookup('env', 'REMOTE_USER') }}"
```
Comment (acozine) [inline ansible/adrl.yml]: Elsewhere we use the variable ansible_ssh_user, please use that here for consistency.

Comment (acozine) [inline ansible/adrl.yml]: oh, never mind - you're using this to set ansible_ssh_user - carry on

File: config/deploy/production.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 set :stage, :production
 set :rails_env, 'production'
 
-server ENV['SERVER'], user: 'deploy', roles: [:web, :app, :db]
+server ENV['SERVER'], user: 'adrl', roles: [:web, :app, :db]
```
Comment (acozine) [inline config/deploy/production.rb]: You'll want the resque_pool role here as well - see https://github.com/curationexperts/alexandria-v2/blob/master/config/deploy/aws.rb#L8

File: config/deploy/aws.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@
 # property set.  Don't declare `role :all`, it's a meta role.
 set :stage, :aws
 set :rails_env, 'production'
-server 'ucsb.curationexperts.com', user: 'deploy', roles: [:web, :app, :db, :resque_pool]
+server 'ucsb.curationexperts.com', user: 'adrl', roles: [:web, :app, :db, :resque_pool]
```
Comment (acozine) [inline config/deploy/aws.rb]: Why are you changing the user on the aws deployment task?

Comment (dunn) [inline config/deploy/aws.rb]: I just changed them all for consistency's sake; if you're using 'deploy' as the user I'll revert that one.

Comment (acozine) [inline config/deploy/aws.rb]: Yes, thanks. You can leave the others, but I want to be sure we can deploy this week uninterrupted on the existing sandbox.

Comment (dunn) [inline config/deploy/aws.rb]: Gotcha; done.

**Pull request merged**